dask-felleskomponenter/src/governance/checks/table.py

```diff
@@ -0,0 +1,191 @@
+from typing import List
+from src.governance.main import TableMetadata
+from .common import MetadataError, Errors
+
```

Comment:
Skal lage et oppslag i funksjonene for feltene som har kodeliste, og sjekke verdi opp mot kodelisten